drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -665,7 +665,7 @@ static int eps_ctx_init(void)
 			err = k_mem_pool_alloc(&ep_buf_pool, &ep_ctx->buf.block,
 					       MAX_EP_BUF_SZ, K_NO_WAIT);
 			if (err < 0) {
-				LOG_ERR("EP buffer alloc failed for EPIN%d", i);
+				LOG_ERR("Buffer alloc failed for EP 0x%02x", i);
```

Comment:
is this endpoint index so we do not know direction?

Reply:
:scream: I will fix it in the next cleanup PR.